android/build.gradle

```diff
@@ -36,6 +36,18 @@ android {
     lintOptions {
         disable 'InvalidPackage'
     }
+    namespace 'com.example.power'
```

Comment:
The namespace is not correct. This namespace is for the android project of the example app not the plugin.